app/core/models/member.py

```diff
@@ -0,0 +1,44 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class Member(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False,
+                          help_text=_("ID único seguindo o padrão UUID4.")
+                          )
+    user = models.ForeignKey('User', on_delete=models.CASCADE, verbose_name=_('usuário'),
+                             help_text=_('Usuário membro de um caderno')
+                             )
+    notebook = models.ForeignKey('Notebook', on_delete=models.CASCADE, verbose_name=_('caderno'),
+                                 help_text=_('Caderno no qual o usuário é membro'),
+                                 )
+
+    class Roles(models.TextChoices):
+        MEMBER = 'member', _('Member')
+        MODERATOR = 'mod', _('Moderator')
+        ADMIN = 'admin', _('Administrator')
+
+    role = models.CharField(max_length=6,
+                            choices=Roles.choices,
+                            default=Roles.MEMBER,
+                            help_text=_("Papel desempenhado pelo membro dentro do caderno")
+                            )
+
+    member_since = models.DateTimeField(auto_now_add=True, verbose_name=_('Data de entrada'),
```

Comment:
`verbose_name` deveria começar por letra minúscula

```suggestion
    member_since = models.DateTimeField(auto_now_add=True, verbose_name=_('data de entrada'),
```